presto-docs/src/main/sphinx/release/release-318.rst

```diff
@@ -0,0 +1,110 @@
+===========
+Release 317
+===========
+
+General Changes
+---------------
+
+* Fix query failure when using ``DISTINCT FROM`` with the ``UUID`` and
+  ``IPADDRESS`` types. (:issue:`1180`)
+* Improve performance for queries over ``information_schema`` that have a limit. (:issue:`999`)
```

Comment:
This should be merged with `Improve performance of information schema tables.` bullet.
```suggestion
```